docs/topics/i18n/translation.txt

```diff
@@ -525,15 +525,18 @@ Localized names of languages
 The ``get_language_info()`` function provides detailed information about
 languages::
 
-    >>> from django.utils.translation import get_language_info
+    >>> from django.utils.translation import get_language_info, activate
+    >>> activate('fr')
     >>> li = get_language_info('de')
-    >>> print(li['name'], li['name_local'], li['bidi'])
-    German Deutsch False
+    >>> print(li['name'], li['name_local, li['name_translated'], li['bidi'])
+    German Deutch Allemand False
 
 The ``name`` and ``name_local`` attributes of the dictionary contain the name of
 the language in English and in the language itself, respectively.  The ``bidi``
 attribute is True only for bi-directional languages.
 
+The ``name_translated`` return language name translated into active language
```

Comment:
Instead of adding a new sentence, I would add the information to the previous paragraph. Something along the lines of:

```
The ``name``, ``name_local``, and ``name_translated`` attributes of the
dictionary contain the name of the language in English, in the language
itself, and in your current active language respectively.  The ``bidi``
attribute is True only for bi-directional languages.
```